modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/util/localizable/transform.es.js

```diff
@@ -28,6 +28,14 @@ export function convertStringToObject(value, localeId) {
 	return value;
 }
 
+export function getISO639LanguageCode(localeId) {
+	if (localeId?.match(/[a-z]{2}_[A-Z]{2}/)) {
+		return localeId.split('_')[0];
+	}
+
+	return localeId;
+}
+
```

Comment:
CKE5 does not like regional dialects, instead using [ISO-639 standard](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes), basically two first letters from our standard locale ID. There might be some edge case that is not covered by the simple regex above, and there might be some customer that does not like this down the line, but this is probably good enough for now.